app/views/users/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= @user.username %></h1>
+
+<h2>Conversations you started: </h2>
+<% @user.conversations.each do |c| %>
```

Comment:
Querying the database is a controllers responsibility. You should do this in your user controller and define it as something like @conversations